examples/angular/src/environments/environment.ts

```diff
@@ -1,5 +1,6 @@
 export const environment = {
+    // Maybe use environment variable to the host URL in the editable-text
     dotcmsUrl: "https://dotcms-qa-master.dotcms.site",
```

Comment:
We have this because we deploy to vercel.

Reply:
Comment removed!